packages/docs/src/pages/getting-started/index.mdx

```diff
@@ -4,13 +4,20 @@ title: Getting Started
 
 # Getting Started
 
-Install Theme UI.
+The `theme-ui` package includes a recommended set of packages out of the box, including [MDX support](/packages/mdx) and a set of [primitive UI components](/packages/components).
```

Comment:
Thanks for updating the core package readme, but I don't think the general *Getting Started* page should branch off from the happy-path installation guide. The idea here is to guide someone new to the library without requiring too many decisions to be made, if that makes sense. I think an addition guide could be added as an advanced use-case, but not necessary for this PR if you don't have time for that

Reply:
Yeah I was 50/50 on that myself, I originally had it as a footnote at the bottom of the first section, but moved and expanded on it as I figured it was a good place to also explain what you get with the main package itself.

For now should I remove that section entirely or move it? I'm happy to work on an advanced guide but I am unsure on how much it could be filled out.

Reply:
I'd say for now, undo the changes in the main getting started page, but feel free to start a stub for a guide or something similar -- doesn't need to be linked to in the nav if it's a WIP